src/db/migrations/migration-2020-08-04-10-00-00-convertTextDataToSql.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const readline = require('readline');
+const mysql = require('mysql2');
+
+const convertDataToSql = () => new Promise(resolve => {
+  const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
+    database: 'todolist',
```

Comment:
Customer may have another credentials to his database (https://www.npmjs.com/package/dotenv).